marketplaceItems/questionApp/appResources/appData/resources/modules/avatarListManager.js

```diff
@@ -0,0 +1,303 @@
+var log = Script.require('https://hifi-content.s3.amazonaws.com/milad/ROLC/d/ROLC_High-Fidelity/02_Organize/O_Projects/Repos/hifi-content/developerTools/sharedLibraries/easyLog/easyLog.js')
```

Comment:
what is this file used for? i can't find any references to it. if it's unused, please remove it.